tools/maxquant/mqwrapper.xml

```diff
@@ -0,0 +1,265 @@
+<tool id="mqwrapper" name="maxquant" version="@VERSION@">
+  <macros>
+    <import>macros.xml</import>
+  </macros>
+  <requirements>
+    <requirement type="package" version="@VERSION@">maxquant</requirement>
+  </requirements>
+  <command detect_errors="exit_code"><![CDATA[
+    python3 $__tool_directory__/mqwrapper.py
+    --tool_dir=$__tool_directory__
+    --num_threads="\${GALAXY_SLOTS:-1}"
+    --raw_files=$raw_files
```

Comment:
You probably want to quote all the files as well. Just imaging one filename/path has a space in it.

So `'$raw_files'` should hopefully work.